lib/shared/get-cache-filename.js

```diff
@@ -1,9 +1,53 @@
+const fs = require('fs');
 const path = require('path');
 
-module.exports = function getCacheFilename(modelConfig) {
+function getCacheFilename(modelConfig, newFile = false) {
     let { slug, cache } = modelConfig;
+    let cacheFiles = [];
+    let cacheFile;
+
+    if (fs.existsSync(cache) && !newFile) {
+        cacheFiles = getCacheFiles(cache)[slug] || [];
+
+        if (cacheFiles.length) {
+            cacheFile = cacheFiles[0].file;
+        }
+    }
 
     return slug && cache
-        ? path.join(cache, `.discoveryjs.${slug}.cache`)
+        ? path.join(cache, cacheFile || `.discoveryjs.${slug}.${Date.now()}.cache`)
```

Comment:
Suggestion: remove `.discoveryjs.` in the beginning and replace `.cache` to `.json`

Reply:
We can have `.` as a cache dir, so we need to somehow determine cache files and ignore them, so in my opinion, we should leave this filename pattern as it is.